README.md

```diff
@@ -1,4 +1,4 @@
-# Expecto TestSdk integration
+# Expecto TestSdk integration [![NuGet Status](http://img.shields.io/nuget/v/YoloDev.Expecto.TestSdk.svg?style=flat)](https://www.nuget.org/packages/YoloDev.Expecto.TestSdk)
```

Comment:
Can you move this down to the other badge(s)?

Reply:
Yes, will do.

Reply:
Done.